src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/whitespace/parenpad/InputParenPadForRecordPattern.java

```diff
@@ -0,0 +1,73 @@
+/*
+ParenPad
+option = (default)nospace
```

Comment:
Please add tests for `space` also

Reply:
Also for regression too.

I see default regression run at https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/83d5013_2024172535/reports/diff/index.html , but nothing with the other `option` property.

Reply:
new test is added and conflicts resolved

Reply:
> but nothing with the other option property.

```
Error:  Java heap space -> [Help 1]
java.lang.OutOfMemoryError: Java heap space
```

looks like it will be difficult to produce report with true option. the amount of violation will be huge

Reply:
Just do checkstyle only.